src/components/common/PeopleAndRoles/CurrentOfficers.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <div id="current-officers">
+    <v-card flat>
+      <!-- Header -->
+      <article class="header-container section-container">
+        <v-icon color="appDkBlue">
+          mdi-account-supervisor
+        </v-icon>
+        <label class="font-weight-bold pl-2">Officers</label>
+        <template v-if="disableLinks">
+          <v-tooltip
+            top
+            color="TextMidGray"
+            content-class="top-tooltip"
+          >
+            <template #activator="{ on, attrs }">
+              <span
+                class="officers-change-wrapper"
+                v-bind="attrs"
+                v-on="on"
+              >
+                <a
+                  class="officers-change disabled-link"
+                  @click.prevent
+                >
+                  <v-icon
+                    medium
+                    class="add-officer-icon"
+                    color="appDkBlue"
+                    v-text="officers.length > 0 ? 'mdi-pencil-outline' : 'mdi-account-plus-outline'"
+                  />
+                  <span
+                    v-if="officers.length > 0"
+                    class="officer-change-text"
+                  >
+                    Manage Officers
```

Comment:
For some reason, this button doesn't clearly show as active when I mouse over it. Compare with the other Change buttons.

Reply:
PS I'm actually looking at the enabled button (not the disabled one in this code).